controllers/spec/checker/SpecChecker.go

```diff
@@ -157,6 +158,11 @@ func (r *SpecChecker) CheckSpec() (SpecCheckResult, error) {
 		specCheckResult.FatalErrorMessage = r.createErrMsgSpecUndefined("spec.image")
 	}
 
+	if spec.Standby.Enabled && spec.Standby.PrimaryEndpoint == emptyStr {
```

Comment:
Could you elaborate? I don't get what you mean.

Reply:
A cluster in stand-by mode is a kubegres cluster that runs a replica that connects to another kubegress cluster.
This mode can be turned on by setting the property `Standby.Enabled` to `true`.
What I am proposing is imagine that we just have one single kubegres cluster in active mode. Imagine that by mistake I set Standby.Enabled` to `true`. That cluster will suddenly have no master node anymore. And I don't know if we can change that back in case we messed it up.
If it is possible that things can be messed up, what I am proposing is to prevent it and not let a kubegress active cluster become a standby.

Reply:
oh I see, yes that make sense, I'll take a look.

Reply:
I'd say to check that in a newer PR, I don't think we have an easy way to do it, as we are not saving the previous state of Kubegres CR.